courses/machine_learning/deepdive2/production_ml/solutions/distributed_training_with_TF.ipynb

```diff
@@ -0,0 +1,1303 @@
+{
```

Comment:
<p>Change the the learning objectives to the following:</p><p> </p><ol><li>Create a MirroredStrategy</li><li>Integrate tf.distribute.Strategy with tf.keras</li><li>Create the input dataset and call tf.distribute.Strategy.experimental_distribute_dataset</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Hi @gstripling,

Updated the learning objectives as per your suggestion.

Thanks.